labs/lab1/chapter1.md

```diff
@@ -182,7 +165,7 @@ the container. Finally, we passed in the name of the image that we built earlier
 Okay, let's make sure we can access the web server.
 
 ```bash
-curl http://cdk.example.com:1080
+curl http://<YOUR AWS INSTANCE PUBLIC DNS NAME HERE>:1080
```

Comment:
it would.  but, they can also hit that from their desktop if they replace with public dns name.  up2u.